### PR TITLE
Login frontend

### LINKED ISSUE #29: Create Login/Registration Frontend

### LINKED ISSUE #37: Project List Page - Authorisation Update
Once authorisation is merged and working, update the project list to require authorisation to view the page.

### LINKED ISSUE #38: Navbar - Authorisation Update
Navbar should update to show Logout button after a user has logged in

### PR DESCRIPTION
**Pull request by pratima-shegokar**
- login form rendered and registration form rendered
- added axios to project dependency
- added apis to connect it to backend
- added new navbar for loggedin user which shows logout button (closes #38 and closes #37 )

this pr closes #29

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -25,3 +25,13 @@
     background-image: url("/Perflow2.2.png");
     font-size: 150%;
 }
+
+.container{
+  vertical-align: middle
+}
+
+#loginLogoDiv {
```
Comment (MelanyCristy) [inline frontend/src/App.css:66]: Same question i asked Jessica. Why did you put this css here? Maybe i'm missing something

File: frontend/src/components/Navbar.js [hunk 2]
```diff
@@ -14,17 +14,14 @@ class Navbar extends React.Component {
                     </button>
                     <div class="collapse navbar-collapse" id="navbarSupportedContent">
                         <ul class="navbar-nav mr-auto">
-                            <li class="nav-item active">
-                                <a class="nav-link" href="/MyProjects">MyProjects <span class="sr-only">(current)</span></a>
```
Comment (MelanyCristy) [inline frontend/src/components/Navbar.js:18]: Why deleting MyProjects click on the navbar?

**Review by jshortz — APPROVED** [review]
Logged in, logged out, and was able to log back in with the same account. Looks good!

File: frontend/src/components/Navbar.js (hunk 2 above)
Comment (jshortz) [reply frontend/src/components/Navbar.js:18]: Because it will only be visible to a logged in user

**Pull request merged**